docs/04.guides/04.cookbooks/29.Threads-Scope/page.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Threads
+id: thread_usage
+---
+## Thread Scope ##
+
+This document explains how to use the threads. Threads mainly used for execute the code parallel.
```

Comment:
***ToChange***
This document explains how to use threads in Lucee. Threads are mainly used for executing code in parallel.